src/aics-image-viewer/components/App/index.tsx

```diff
@@ -418,7 +482,7 @@ const App: React.FC<AppProps> = (props) => {
       aimg = await loader.current.createVolume(loadSpec, (v, channelIndex) => {
         // NOTE: this callback runs *after* `onNewVolumeCreated` below, for every loaded channel
         // TODO is this search by name necessary or will the `channelIndex` passed to the callback always match state?
-        const thisChannelSettings = getOneChannelSetting(v.imageInfo.channelNames[channelIndex]);
+        const thisChannelSettings = viewerState.current.channelSettings[channelIndex];
```

Comment:
Had to change channels from looking up the settings by name to using the channel's index in a few places to fix functionality in CFE; I suspect this was caused by the change to `ViewerStateProvider` and the consolidation of the channel states into an array.

Reply:
I'm honestly not sure, I think there's only one place that's still using `getOneChannelSetting` remaining after this PR. @frasercl can you weigh in? I'm seeing a TODO immediately above this on line 420 that is asking this as well.